command/operator_debug.go

```diff
@@ -246,6 +246,13 @@ func (c *OperatorDebugCommand) Run(args []string) int {
 	}
 	c.interval = i
 
+	// Sanity check interval
```

Comment:
Although it's unfortunately used a few places still, "sanity" probably isn't a term we'd like to use in comments.

```suggestion
	// Validate interval
```

Reply:
Excellent point -- fixed.